apps/site/src/views/Resources/sections/StarterPacks/StarterPacks.tsx

```diff
@@ -20,13 +20,13 @@ async function StarterPacks() {
 			</div>
 			<div className={styles["bottom-spacer"] + " row"}>
 				{resources.map(
-					({ _id, title, description, link, logo, stickyNoteColor }) => (
+					({ _id, title, description, link, logo, background }) => (
 						<div className={styles.column + " col"} key={_id}>
 							<ResourceCard
 								title={title}
 								description={<PortableText value={description} />}
 								stickerSrc={urlImageBuilder(client).image(logo).url()}
-								stickyNoteColor={stickyNoteColor.hex}
+								islandBackground={urlImageBuilder(client).image(background).url()}
```

Comment:
Issue: This block seems to be a bit repetitive since it appears in multiple components. Would it be possible to extract it to a common functions file? Additionally, `ApiResources` uses the `key` attribute while the others don't, so it would be good to choose one way or the other and stick with it.

Reply:
I think this might be an issue for another separate reorganization pr (#171 suggests this too so eventually we will implement but maybe not here)